src/main/ngapp/accio/src/app/img-gallery/img-gallery.component.ts

```diff
@@ -132,10 +134,166 @@ export class ImgGalleryComponent implements OnInit {
     dialogRef.afterClosed().subscribe(() => {
       console.log('UpdateImage dialog was closed...');
       console.log('Fetching updated images...');
-      this.getImages();
+      this.loadGalleryImages();
       console.log('Fetched updated images...');
     });
   }
+
+  /** 
+   * Fetch images with parameters specified by class members.
+   */
+  async fetchImages(): Promise<any> {
```

Comment:
Here and below it would be good to use a more stricter type than any.

You can add a todo and follow up later if appropriate.  This is good to submit.

Reply:
Done